{{cookiecutter.project_name}}/.mlops-setup-scripts/README.md

```diff
@@ -112,13 +128,20 @@ on the current repo.
 This token is used to:
 1. Fetch ML code from the current repo to run on Databricks for CI/CD (e.g. to check out code from a PR branch and run it
```

Comment:
Nit: It's a bit weird to see this render as a single-item list:

<img width="453" alt="image" src="https://user-images.githubusercontent.com/2358483/196308660-1b795fe7-ebe0-4268-ac88-78031324043f.png">


Maybe we can just if/else the whole block so that it looks something like this:

```
The setup script prompts a Git token with both read and write permissions
on the current repo.

This token is used to fetch ML code from the current repo to run on Databricks for CI/CD (e.g. to check out code from a PR branch and run it
during CI/CD). You can generate a PAT token for Azure DevOps by following the steps described [here](https://learn.microsoft.com/en-us/azure/devops/organizations/accounts/use-personal-access-tokens-to-authenticate?view=azure-devops&tabs=Windows).
```

Reply:
agreed, and fixed